server/src/services/boards/columns.go

```diff
@@ -8,23 +8,37 @@ import (
 	"github.com/google/uuid"
 	"scrumlr.io/server/common"
 	"scrumlr.io/server/common/dto"
+	"scrumlr.io/server/common/filter"
 	"scrumlr.io/server/realtime"
 
 	"scrumlr.io/server/database"
 	"scrumlr.io/server/logger"
 )
 
-func (s *BoardService) CreateColumn(_ context.Context, body dto.ColumnRequest) (*dto.Column, error) {
+func (s BoardService) CreateColumn(_ context.Context, body dto.ColumnRequest) (*dto.Column, error) {
```

Comment:
I don't think its relevant if the receiver is a pointer or value but we should keep the consistency by having all receivers be a pointer or value